docs/SchemaMigrationTool.md

```diff
@@ -0,0 +1,49 @@
+# Schema Migration Tool
+The Schema migration tool is the command line utility to perform schema upgrade on demand. The tool will automatically identify current version of the database in order to apply appropriate migration scripts.
+
+Note - The tool can't downgrade a schema version.
+
+- #### Prerequisites
+    The tool would only run when the current schema is in conformance to the [BaseSchema](BaseSchema.md). So either the BaseSchema should already be present or if not, then Schema admin would need to run BaseSchema script manually.
+
+- #### How to install the tool
+
+    - ##### If available as .NET Core global tool 
+
+        It can be installed like this:
+
+            Open a terminal/command prompt 
+            Type 'dotnet tool install -g Microsoft.Health.SchemaManager'
```

Comment:
This section is being displayed like it was code. I don't think this is the right style for the section.